src/core/core.datasetController.js

```diff
@@ -65,6 +135,15 @@ module.exports = function(Chart) {
 			this.update(true);
 		},
 
+		/**
+		 * @private
+		 */
+		destroy: function() {
+			if (this._data) {
+				unlistenArrayEvents(this._data, this);
+			}
+		},
+
 		createMetaDataset: function() {
```

Comment:
I realized this method isn't mentioned in the docs ... is that intentional? We don't document the `datasetElementType` and `dataElementType` properties either